public/src/components/drawer.tsx

```diff
@@ -210,6 +210,11 @@ export default function NavDrawer(): React.ReactElement {
             <span className={classes.super}>🎨</span>
           </ListItem>
         </Link>
+        <Link key="Support Landing Page" to="/support-landing-page-tests" className={classes.link}>
```

Comment:
we've decided to keep the tool hidden from the menu for now, so lets remove this before merging